src/main/java/com/backbase/ct/bbfuel/data/ProductSummaryDataGenerator.java

```diff
@@ -88,7 +101,8 @@ private static String getProductTypeNameFromProductsInputFile(String productId)
             String currentAccountName = productGroupSeed.getCurrentAccountNames().get(currentAccountNameIndex);
             String currency = productGroupSeed.getCurrencies().get(currencyIndex);
             ArrangementsPostRequestBody arrangementsPostRequestBody = getArrangementsPostRequestBody(
-                externalLegalEntityId, currentAccountName, currency, 1);
+                Optional.ofNullable(staticCurrentaccountArrangementsQueue.poll()), externalLegalEntityId,
```

Comment:
Can we instead add the static arrangements before or after adding the random ones?

Reply:
This code adds them before the random ones

Reply:
It's giving static IDs to arrangements that would otherwise have random ones. I mean adding _more_ arrangements using static IDs after those ones.

Reply:
Sorry, I didn't get this

Reply:
Discussed offline: this would make the number of arrangements created not match `numberOfArrangements` which could be confusing, so we're not doing it.